content/docs/reference-events.md

```diff
@@ -26,7 +26,6 @@ void preventDefault()
 boolean isDefaultPrevented()
 void stopPropagation()
 boolean isPropagationStopped()
-void persist()
```

Comment:
Technically it's still there so it should be in the API reference. Just needs to note it doesn't do anything.

Reply:
@gaearon 
Thank you. I've added the note as another note at https://github.com/reactjs/reactjs.org/pull/3207/commits/89a9c297799683b4bf159774f03c0c2178438732.